src/random-seed/random-seed.c

```
@@ -122,6 +126,39 @@ static int parse_argv(int argc, char *argv[]) {
         return 1;
 }
 
+static void write_entropy_xattr(int fd, int entropy_count)
+{
+        char buf[32];
+        int len;
+
+        len = snprintf(buf, sizeof(buf), "%d", entropy_count);
+        if (fsetxattr(fd, XATTR_NAME, buf, len+1, 0)) {
+                if (errno != ENOTSUP)
+                        log_warning_errno(errno, "Failed to set xattr '%s'", XATTR_NAME);
+                /* If we fail for any reason, attempt to delete the xattr. We
+                 * can't do much if that fails as well. */
+                fremovexattr(fd, XATTR_NAME);
```

please cast this to `(void)` to tell static analyzers we knowingly ignore errors from this syscall